Editor/Window/ManagedEC2Page.cs

```diff
@@ -151,12 +156,21 @@ private void UpdateGUI()
             }
             else if (stackStatus.IsStackStatusFailed())
             {
-                _statusIndicator.Set(State.Failed, textProvider.Get(Strings.ManagedEC2DeployStatusFailed));
+                _statusIndicator.Set(State.Failed, textProvider.GetError(ErrorCode.StackStatusInvalid));
+                _deployStatusBox.Show(StatusBox.StatusBoxType.Error, _deploymentSettings.CurrentStackInfo.Details);
```

Comment:
```suggestion
                _statusIndicator.Set(State.Failed, textProvider.Get(Strings.ManagedEC2DeployStatusFailed));
                _deployStatusBox.Show(StatusBox.StatusBoxType.Error, textProvider.GetError(ErrorCode.StackStatusInvalid));
```

The indicator should be just `Failed` not a sentence, while the status box should display the sentence (identical to rolled back/rolling back)